theano/gof/opt.py

```diff
@@ -201,7 +195,12 @@ def warn(exc, self, optimizer):
 
     def __init__(self, *opts, **kw):
         """
-        WRITEME
+        Parameters
+        ----------
+        opts : List
+        The List of optimizers to be applied to a node
+        kw : Dict
```

Comment:
Here you should document the supported keyword args and mention that they are keyword-only.